fitbenchmarking/controllers/bumps_controller.py

```diff
@@ -11,6 +11,20 @@
 from fitbenchmarking.cost_func.cost_func_factory import create_cost_func
 from fitbenchmarking.utils.exceptions import CostFuncError
 
+ALGORITHM_CHECK = {
```

Comment:
As mentioned in the sphinx extension, I think these would be better defined within the class (ideally before the init)